tests/unit/notice_transformer/conftest.py

```diff
@@ -13,7 +14,32 @@ def fake_rml_mapper() -> RMLMapperABC:
 
 
 @pytest.fixture
-def fake_mapping_suite() -> MappingSuite:
-    repository_path = TEST_DATA_PATH / "notice_transformer"
+def fake_mapping_suite_id() -> str:
+    return "test_package"
+
+
+@pytest.fixture
+def fake_not_mapping_suite_id() -> str:
+    return "test_not_package"
+
+
+@pytest.fixture
+def fake_failed_mapping_suite_id() -> str:
+    return "test_failed_package"
+
+
+@pytest.fixture
+def fake_repository_path() -> Path:
+    return TEST_DATA_PATH / "notice_transformer" / "mappings"
+
+
+@pytest.fixture
+def fake_mapping_suite(fake_repository_path, fake_mapping_suite_id) -> MappingSuite:
+    repository_path = fake_repository_path
     mapping_suite_repository = MappingSuiteRepositoryInFileSystem(repository_path=repository_path)
-    return mapping_suite_repository.get(reference="test_package")
+    return mapping_suite_repository.get(reference=fake_mapping_suite_id)
+
+
+@pytest.fixture
+def cmd_transformer_path() -> str:
+    return "../../../../ted_sws/notice_transformer/entrypoints/cmd_mapping_suite_transformer.py"
```

Comment:
This approach is suboptimal, and using a relative path like that is not safe at all. 
It is much more elegant, easy and safe if one employs the testing facilities offered by the `Click` library. See the testing section in the documentation.